lib/screens/widgets/project_widget.dart

```diff
@@ -47,8 +49,8 @@ class ProjectWidget extends StatelessWidget {
               child: ElevatedButton(
                 onPressed: () async {
                   //Launch project on GitHub
-                  final Uri _url = Uri.parse(projectData.link);
-                  await launchUrl(_url);
+                  final Uri url = Uri.parse(projectData.link);
```

Comment:
Was this change required? Don't understand why private `_url` had to be changed to `url`.